README.md

```diff
@@ -1,2 +1,367 @@
+$ OCP_VIRT_VALIDATION_IMAGE=$(oc get csv -n openshift-cnv $CSV_NAME -o json | jq -r '.spec.relatedImages[] | select(.name | contains("ocp-virt-validation-checkup")).image)'
+```
+
+### Run the validation checkup
+Once the OCP-Virt validation checkup image is obtained, run it to generate the manifests required to run the checkup.
+Dump the run manifests to stdout:
+```bash
+$ podman run -e OCP_VIRT_VALIDATION_IMAGE=${OCP_VIRT_VALIDATION_IMAGE} ${OCP_VIRT_VALIDATION_IMAGE} generate
+```
+This will generate the following manifests:
+```yaml
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ocp-virt-validation
+spec: {}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ocp-virt-validation-sa
+  namespace: ocp-virt-validation
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: ocp-virt-validation-cluster-admin-binding
+subjects:
+  - kind: ServiceAccount
+    name: ocp-virt-validation-sa
+    namespace: ocp-virt-validation
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ocp-virt-validation-pvc-20250520-105358
+  namespace: ocp-virt-validation
+  labels:
+    app: ocp-virt-validation
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: ocp-virt-validation-job-20250520-105358
+  namespace: ocp-virt-validation
+spec:
+  template:
+    metadata:
+      labels:
+        app: ocp-virt-validation
+    spec:
+      serviceAccountName: ocp-virt-validation-sa
+      securityContext:
+        fsGroup: 1001
+      containers:
+        - name: ocp-virt-validation-checkup
+          image: registry.redhat.io/container-native-virtualization/ocp-virt-validation-checkup-rhel9:v4.19.0-28
+          imagePullPolicy: Always
+          env:
+            - name: DRY_RUN
+              value: "false"
+            - name: TIMESTAMP
+              value: 20250520-105358
+            - name: RESULTS_DIR
+              value: /results
+            - name: TEST_SUITES
+              value: compute,network,storage,ssp
+            - name: TEST_SKIPS
+              value: ""
+          volumeMounts:
+            - name: results-volume
+              mountPath: /results
+      restartPolicy: Never
+      volumes:
+        - name: results-volume
+          persistentVolumeClaim:
+            claimName: ocp-virt-validation-pvc-20250520-105358
+  backoffLimit: 0
+```
+In order to apply these manifests directly onto the cluster in order to start the validation checkup, run:
+```bash
+podman run -e OCP_VIRT_VALIDATION_IMAGE=${OCP_VIRT_VALIDATION_IMAGE} ${OCP_VIRT_VALIDATION_IMAGE} generate | oc apply -f -
+```
+You can funnel the output to a file in order to make adjustments if needed:
```

Comment:
```suggestion
You can tunnel the output to a file in order to make adjustments if needed:
```

Reply:
actually i meant "funnel" for the meaning of outputing into a file.
but looks like the official term for it is "output redirection"